src/index.js

```diff
@@ -328,9 +327,6 @@ async function watch(stateManager, mainIndex) {
 
 async function error(err) {
   sentry.report(err);
```

Comment:
we can pass more info here wrt the json problem, but I couldn't find where to get the original json